modules/installation-restricted-network-samples.adoc

```diff
@@ -3,7 +3,11 @@
 // * installing/install_config/installing-restricted-networks-preparations.adoc
 // * openshift_images/samples-operator-alt-registry.adoc
 
-ifeval::["{context}" == "installing-restricted-networks-preparations"]
+ifeval::["{context}" == "installing-mirroring-installation-images"]
```

Comment:
Does _ installing/install_config/installing-restricted-networks-preparations.adoc_ need to be updated to _post_installation_configuration/cluster-tasks_?